src/Resources/views/standard_layout.html.twig

```diff
@@ -68,14 +68,14 @@ file that was distributed with this source code.
             {# localize moment #}
             {% set localeForMoment = canonicalize_locale_for_moment() %}
             {% if localeForMoment %}
-                <script src="{{ asset('bundles/sonataform/moment-locale/' ~ localeForMoment ~ '.js') }}"></script>
+                <script src="{{ asset('bundles/sonataform/moment-locale/' ~ localeForMoment ~ '.js?v=2.0') }}"></script>
             {% endif %}
 
             {# localize select2 #}
             {% if sonata_config.getOption('use_select2') %}
                 {% set localeForSelect2 = canonicalize_locale_for_select2() %}
                 {% if localeForSelect2 %}
-                    <script src="{{ asset('bundles/sonataadmin/select2-locale/' ~ localeForSelect2 ~ '.js') }}"></script>
+                    <script src="{{ asset('bundles/sonataadmin/select2-locale/' ~ localeForSelect2 ~ '.js?v=4.36') }}"></script>
```

Comment:
Not sure we need versioning for a dependency of sonata admin, the main issue is about our own js